website_sale_line_total/views/website_sale_line_total_view.xml

```diff
@@ -27,4 +27,39 @@
         </xpath>
     </template>
 
+    <template id="add_payment_item_line_total" inherit_id="website_sale.payment">
+        <xpath expr="//table[@id='cart_products']/tbody/tr/td[span[contains(@t-field,'line.price_reduce_tax')]]" position="attributes">
```

Comment:
not sure this is correct: 
1. it selects multiple elements
2. it applies an ID to something that is within a loop -> it should be a class

Reply:
OMG, they did it wrong https://github.com/odoo/odoo/blob/10.0/addons/website_sale/views/templates.xml#L868
Not a good reason to replicate the error tho... no? :)